calico-cloud/image-assurance/scanners/pipeline-scanner.mdx

```diff
@@ -2,23 +2,23 @@
 description: Scan images in your build pipeline using Image Assurance.
 ---
 
-# Integrate Image Assurance scanner in your build pipeline
+# Integrate scanner into your build pipeline
 
 ## Big picture
 
-Integrate the Image Assurance scanner into your build pipeline and private registries to ensure builds are checked by Image Assurance before deployment. You can also use the scanner offline and on-demand for ad hoc scanning and emergency patching. For a real-world use case, see [Hands-on guide: How to scan and block container images to mitigate SBOM attacks](https://www.tigera.io/blog/hands-on-guide-how-to-scan-and-block-container-images-to-mitigate-sbom-attacks/).
+Integrate the Image Assurance scanner into your build pipeline to ensure builds are checked by Image Assurance before deployment.
 
-This article describes how to install the Image Assurance binary. To integrate the containerized version of Image Assurance scanner into your CI/CD platform, go to: [Image Assurance containerized scanner](https://quay.io/repository/tigera/image-assurance-scanner-cli) and pull the latest image. For example:
+## Value
 
-```bash
-   docker pull quay.io/tigera/image-assurance-scanner-cli:v1.8.4
-```
+The Image Assurance CLI scanner allows you to manually scan container images locally or remotely for on-demand scanning and emergency scanning. The CLI scanner is ideal for use in a CI/CD pipeline to automatically scan images before pushing them to a registry.
+
+Scanning is done before runtime and the results are displayed in the Image Assurance dashboard in Manager UI. You can then use [Image Assurance Admission Controller](../install-the-admission-controller) to automatically blocks resources that would create containers with vulnerable images from entering your cluster. For a real-world use case, see [Hands-on guide: How to scan and block container images to mitigate SBOM attacks](https://www.tigera.io/blog/hands-on-guide-how-to-scan-and-block-container-images-to-mitigate-sbom-attacks/).
 
 ## Before you begin
 
 **Image requirements**
 
-Images must be available locally through the container runtime environment where the Image Assurance scanner is running
+Images must be available locally through the container runtime environment where the Image Assurance scanner is running.
```

Comment:
Here we should clarify that we only support `docker` for the CLI.

Reply:
Fixed.